### PR TITLE
Add option for disabling generation of default keystore password

### PR DESCRIPTION
**Pull request by aguibert**
Per the discussion of #1627 users should be able to opt-out of default password generation using:
`bin/server create myServer --no-password`

### CODE REVIEW

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#build

**Comment by mhldr** [comment]
issues discussed in #was-liberty-ops caused the build to fail so I'm respinning this incase it's needed (you ***may*** already have the results you need but if not hopefully this respin will help)
#build

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#build

File: dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/BootstrapConfig.java [hunk 1]
```diff
@@ -1165,18 +1166,23 @@ public String run() throws Exception {
         try {
             char[] keystorePass = PasswordGenerator.generateRandom();
```
Comment (brenthdaniel) [inline dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/BootstrapConfig.java]: generateRandom could be moved into the if block

**Pull request merged**